### ISSUE TITLE
Hovering and moving the mouse over an AlphaMask interrupts function-driven transforms and some atl

### ISSUE THREAD

**Issue by Gouvernathor** [renpy/renpy#2458]
Pretty technical here.
When a displayable is an AlphaMask or includes an AlphaMask (through some sort of Container or udd), and when a displayable or displayables use a `Transform(function=function)` transform, the said function(s) are all kept from executing when the user _moves_ their mouse over the alphamask-ed displayable (hovering it with a still mouse is fine).
Example code :
```
init python;
    def blink(trans, st, at):
        print(st)
        if renpy.random.random() < .5:
            trans.alpha = .9
        else:
            trans.alpha = 1
        return 0

show expression Transform('eileen happy', function=blink) as eright at right
pause
show expression AlphaMask('eileen happy', Solid('#fff')) as eleft at left
pause
show expression Transform(AlphaMask('eileen happy', Solid('#fff'), function=blink) as emid at default
pause
```
Moving the mouse over the left or middle ones inhibits the blinking effect of the middle and right ones.
Adding the `print` lags the game but shows (with the "Console output" launcher option) that the function isn't even triggered.

I strongly believe it's _not_ related to the `event` method of AlphaMask being triggered since it's the same as renpy.display.layout.Container and Container doesnt trigger this glitch.

**Comment by Gouvernathor** [comment]
Actually it also inhibits alt, or at least some transforms defined with atl. For example a displayable I had also froze when I moved my mouse over an AlphaMask, and the atl applied to the displayable was the following
```
transform wave(range, toffset=0.0, speed=.6):
    subpixel True
    pause toffset
    easein speed/2 yoffset range
    block:
        ease speed yoffset -range
        ease speed yoffset range
        repeat
```

**Comment by renpytom** [comment]
Works for me. 

I'll note that your example doesn't work - 

```
show expression Transform(AlphaMask('eileen happy', Solid('#fff'), function=blink) as emid at default
```

Seems to be missing a parenthesis. What I tested with was:

```
init python:
    def blink(trans, st, at):
        # print(st)

        if renpy.random.random() < .5:
            trans.alpha = .9
        else:
            trans.alpha = .1

        return 0

label start:

    scene bg washington

    show expression Transform('eileen happy', function=blink) as eright at right
    show expression AlphaMask('eileen happy', Solid('#fff')) as eleft at left
    show expression Transform(AlphaMask('eileen happy', Solid('#fff')), function=blink) as emid at default
    pause
```

I wouldn't expect this to fail, and as far as I can tell, it's not.

**Comment by Gouvernathor** [comment]
I tested that corrected code on a new dummy project, with a clean Nightly install. The bug happens.
https://www.youtube.com/watch?v=3AzuTgrOmAw
You can check the same code is used by pausing the start of the video.

**Comment by Gouvernathor** [comment]
@renpytom I tested that corrected code on a new dummy project, with a clean Nightly install. The bug happens.
https://www.youtube.com/watch?v=3AzuTgrOmAw
You can check the same code is used by pausing the start of the video.

**Comment by renpytom** [comment]
I still can't repeat this with the 2020-11-18 nightly, and there isn't a plausible thing I can think of that would cause this to occur. If you want to package your game up, I can try it. Beyond that, I'm not sure what I can do.

**Comment by Gouvernathor** [comment]
I just tried packaging it : in the exported version the bug appears on the computer where I'm coding, but not on a friend's computer.
I then tried what I hadn't thought of before : to change my default GPU, setting it on an nVidia card instead of the integrated graphics (which means my CPU I guess) disables the glitch.
I really don't know if my "integrated graphics" is half-baked or if it's a problem with renpy's graphics engine (or whatever) not communicating properly with it.
As long as nobody else reports this glitch, I'll consider it's the first option, but as it appeared when I went from 7.3.5 to 7.4, it may happen elsewhere when 7.4 will be released as the stable version.
